tests/phpunit/tests/test-admin-filters-term.php

```diff
@@ -755,4 +755,15 @@ public function test_update_term_slug() {
 
 		unset( $_REQUEST, $_POST );
 	}
+
+	/**
+	 * @ticket #694 {@see https://github.com/polylang/polylang-wc/issues/694}
+	 */
+	public function test_create_term_with_empty_slug_and_already_existing_name() {
+		$en = self::factory()->term->create( array( 'taxonomy' => 'category', 'name' => 'Cats', 'slug' => 'cat' ) );
```

Comment:
```suggestion
		self::factory()->term->create( array( 'taxonomy' => 'category', 'name' => 'Cats', 'slug' => 'cat' ) );
```
To fix last PHPCS error.